docs/integrations/index.rst

```diff
@@ -1,24 +1,84 @@
+==========================================
 Integrations
 ==========================================
 
 .. _SEP-24: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md
 .. _Django Commands: https://docs.djangoproject.com/en/2.2/howto/custom-management-commands
 .. _stellar-anchor-server: https://github.com/stellar/stellar-anchor-server
 
-Polaris does most of the work implementing SEP-24_. However, Polaris simply
-doesn't have the information it needs to interface with an anchor's partner
-financial entities. This is where :class:`.DepositIntegration` and
-:class:`.WithdrawalIntegration` come in.
+Polaris does most of the work implementing SEP-24_. However, some pieces of
+SEP-24 can only be implemented by the anchor. Specifically, anchors need to
+implement their own banking rails and KYC requirements. This is where
+:class:`DepositIntegration` and :class:`WithdrawalIntegration` come in.
```

Comment:
"This is where the Integrations classes come in" since KYC is added now and Deposit/WithdrawIntegration doesn't cover all the integrations